src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -183,14 +190,15 @@ export async function $onEmit(context: EmitContext<NetEmitterOptions>) {
         csharpGeneratorPath: resolvedOptions.csharpGeneratorPath,
         "clear-output-folder": resolvedOptions["clear-output-folder"],
         "save-inputs": resolvedOptions["save-inputs"],
-        "model-namespace": resolvedOptions["model-namespace"]
+        "model-namespace": resolvedOptions["model-namespace"],
+        "generate-all-models": resolvedOptions["generate-all-models"]
```

Comment:
how does this flag differ from this flag [here](https://github.com/Azure/azure-sdk-for-net/pull/34058/files#diff-e8ccddd90220fbc6a949079f96ecfac074affbeb698e4c1bb8a3f850445342ceR114).